plugins/search-react/api-report.md

```diff
@@ -118,6 +147,7 @@ export const SearchContextProvider: (
 // @public
 export type SearchContextProviderProps = PropsWithChildren<{
   initialState?: SearchContextState;
+  parentContext?: boolean;
```

Comment:
Would be good to call this new prop out with a separate changeset on the `search-react` package as well.